src/core_plugins/kibana/public/management/sections/settings/components/page_title.js

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
we've been following a convention where each component has its own directory, this allows for nesting sub components if needed and keeps test files and snapshots bundled together. I know this is a very simple component, but would like to see the same convention followed 🙂

also would like to see a jest test. and since the page title has been pulled out of `advanced_settings.js`, that file's tests/snapshots need to be updated